### ISSUE TITLE
[WIP] Master Accessibility Ticket

### ISSUE THREAD

**Issue by youriwims** [mozilla/foundation.mozilla.org#3934]
## CI Testing
1. Enable A11y CI 
    1.  Integrate `react-axe` / `cypress-axe` with Foundation repo for local testing [**#4058**]
    2.  Add `axe-cli` to Foundation site CI [ **#4059**]

## Pages in need of some TLC*
  1. Global
     1. Nav [**#4001**]
     2. Footer [**#4002**]
  2. Fo.Mo Blog
        1. Blog Post [**#4004**]
        2. Blog Homepage [**#4003**]
  3. PNI
       1. Home [**#4005**]
       2. Product [**#4006**]
       3. About [**#4007**]
  4. Fo.Mo
      1. About [**#4009**]
      2. Home [**#4008**]
      3. Initiatives
      4. Participate
      5. Internet Health
      6. Fellowships
---
**Key actions pp** **
      

  - Semantic Markup
    - Proper Outline
  - Use html tags
     - If unable, find proper aria-* or styling.
     - Each page should have a unique, descriptive h1 and page title
   - Colour contrast
     - Make sure works in b/w
     - Check in high contrast
   - Keyboard Navigation
      - Elements should be focusable
      - Other manual testing
   - Markup Validation
   - Keyboard events for newsletter signup forms 

_* Organized based on most visited/navigation summary from GA results._
_** In addition to/ an extension of standard browser extension testing._

**More details to come**

**Comment by jamilasnell** [comment]
We should close this ticket and do a re-evaluation of this work so that we can be up to date!
